extension/encoding/awslogsencodingextension/internal/unmarshaler/elb-access-log/testdata/nlb_al_valid_logs_expected.yaml

```diff
@@ -37,15 +37,87 @@ resourceLogs:
               - key: aws.elb.tls.listener.resource_id
                 value:
                   stringValue: g3d4b5e8bb8464cd
+              - key: aws.elb.connection_time
+                value:
+                  intValue: "5"
+              - key: aws.elb.tls_handshake_time
+                value:
+                  intValue: "2"
+              - key: aws.elb.tls_connection_creation_time
+                value:
+                  stringValue: 2018-12-20T02:59:30
+              - key: aws.elb.chosen_cert_arn
+                value:
+                  stringValue: arn:aws:acm:us-east-2:671290407336:certificate/2a108f19-aded-46b0-8493-c63eb1ef4a99
+              - key: tls.cipher
+                value:
+                  stringValue: ECDHE-RSA-AES128-SHA
               - key: tls.protocol.version
                 value:
                   stringValue: tlsv12
+              - key: url.domain
+                value:
+                  stringValue: my-network-loadbalancer-c6e77e28c25b2234.elb.us-east-2.amazonaws.com
+            body: {}
+            timeUnixNano: "1545274780000000000"
+          - attributes:
+              - key: network.protocol.name
+                value:
+                  stringValue: tls
+              - key: network.protocol.version
+                value:
+                  stringValue: "2.0"
+              - key: client.address
+                value:
+                  stringValue: 72.21.218.154
+              - key: client.port
+                value:
+                  intValue: "51341"
+              - key: destination.address
+                value:
+                  stringValue: 172.100.100.185
+              - key: destination.port
+                value:
+                  intValue: "443"
+              - key: http.request.size
+                value:
+                  intValue: "98"
+              - key: http.response.size
+                value:
+                  intValue: "246"
+              - key: aws.elb.tls.listener.resource_id
+                value:
+                  stringValue: g3d4b5e8bb8464cd
+              - key: aws.elb.connection_time
+                value:
+                  intValue: "5"
+              - key: aws.elb.tls_handshake_time
+                value:
+                  intValue: "2"
+              - key: aws.elb.tls_connection_creation_time
+                value:
+                  stringValue: ',"http/1.1"'
```

Comment:
This doesn't seem right. Can you check @Kavindu-Dodan ?

Description for this field is:

> tls_connection_creation_time: The time recorded at the beginning of the TLS connection, in ISO 8601 format.

So something must be off in the code

Reply:
Ahh, first of all good catch 👏

Yup will be painful given it's string parsing 😿  will see what I can do about this.

Reply:
I found a good fix for this. Will update PR soon.

Reply:
@constanca-m fixed this with commit - https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/44109/commits/5d05a90048f87ff8eac858b008bb8ff4a1e35bed

Also added unit tests :)